Documentation/doc/biblio/geom.bib

```diff
@@ -152085,3 +152086,33 @@ @article{ledoux2014triangulation
   year={2014},
   publisher={Elsevier}
 }
+
+@article{schonhardt1928zerlegung,
+  title={{\"U}ber die zerlegung von dreieckspolyedern in tetraeder},
+  author={Sch{\"o}nhardt, Erich},
+  journal={Mathematische Annalen},
+  volume={98},
+  number={1},
+  pages={309--312},
+  year={1928},
+  publisher={Springer}
+}
+
+@inproceedings{si2005meshing,
+  title={Meshing piecewise linear complexes by constrained Delaunay tetrahedralizations},
+  author={Si, Hang and G{\"a}rtner, Klaus},
+  booktitle={Proceedings of the 14th international meshing roundtable},
```

Comment:
```suggestion
  booktitle={Proceedings of the 14th international Meshing Roundtable},
```

Reply:
I saw this warning as well.
I think this a side effect of the merging strategy used by CGAL for the overnight builds, so I didn't do a followup on it.

Reply:
That was my guess too. (I did the merge while waiting at the blood donation yesterday so conflict resolution was not optimal :))